src/App.js

```diff
@@ -0,0 +1,3 @@
+const heading2 = React.createElement("h1", null, "Hello world using React");
+const root2 = ReactDOM.createRoot(document.getElementById("root2"));
```

Comment:
naming of variables is not good. Please avoid digits in variables. Keep it root only. variables should define the action  it is used for.